src/app/services/ccd-config/ccd-case.config.ts

```diff
@@ -1,7 +1,13 @@
-import {Injectable} from '@angular/core';
-import {AbstractAppConfig, CaseEditorConfig} from '@hmcts/ccd-case-ui-toolkit';
-import {AppConfigService} from '../config/configuration.services';
-
+import { Injectable } from '@angular/core';
+import {
+  AbstractAppConfig,
+  CaseEditorConfig,
```

Comment:
Any reason why these imports have wrapped? They shouldn't exceed 140 characters on a single line.

Reply:
Also, there's a trailing comma.

Reply:
Prettify did it for me. Fixed it now. Removed comma